driver-core/src/main/com/mongodb/connection/ConnectionId.java

```diff
@@ -38,8 +38,9 @@ public final class ConnectionId {
     private static final AtomicInteger INCREMENTING_ID = new AtomicInteger();
```

Comment:
It's internal, but for clarity should we also change this to `AtomicLong`?

Reply:
Thanks, we should update it.

Done.